conf/empire/example.env

```diff
@@ -1,3 +1,4 @@
+namespace: empire.example
```

Comment:
Lets remove the default here - the reason is that since the namespace is used for the s3 bucket name, this'll likely cause confusion.  Give a good comment to to explain what people should set this too.